### ISSUE TITLE
the lack of open world assets

### ISSUE THREAD

**Issue by hamedsheygh** [TheGameCreators/GameGuruRepo#1801]
I think we need some dead and rusty cars in our games that based on open world type. I feel the lack of post-apocalypse assets in the end of design result.

**Comment by casigus** [comment]
they have them for sale on the store or you can find some on sketchfab (must give full credit to the model maker, and there are lots of ones there you can use)

**Comment by hamedsheygh** [comment]
NO,casigus. After 15 years of game engine development we need some basic high quality 3d models that cover the area, and this is NOT our duty, this is game guru max developers duty. We MUST NOT pay for every single basic object in the map, we need some BASICS for the release time.

**Comment by synchromesh62** [comment]
@hamedsheygh 
Your not paying for every single object.
GameGuruMax comes with over 1000 Royalty free assets which is more than any other engine available on steam. Max is a Game Creation tool and has a built in importer for you to import your own model creations or 3rd party assets. It is most certainly not the duty of the developers to supply them.

**Comment by hamedsheygh** [comment]
@synchromesh62 
WOW,thats amazing this is My first time that i  heard about that. (thank you) but i still wait for it.
